docs/references/errors.rst

```diff
@@ -3,7 +3,7 @@
 Errors
 ######
 
-PostgREST error messages follow the PostgreSQL error structure. It includes ``MESSAGE``, ``DETAIL``, ``HINT``, ``ERRCODE`` and will add an HTTP status code to the response.
+PostgREST error messages follow the PostgreSQL error structure. It includes ``MESSAGE``, ``DETAIL``, ``HINT``, ``ERRCODE`` and will add an HTTP status code to the response. Since *PostgREST 11.3.0*, you can add HTTP headers to the response as well.
```

Comment:
I think this is not necessary. We can better highlight the feature on the release page like we did for "domain representations": https://github.com/PostgREST/postgrest/releases/tag/v11.2.0

```suggestion
PostgREST error messages follow the PostgreSQL error structure. It includes ``MESSAGE``, ``DETAIL``, ``HINT``, ``ERRCODE`` and will add an HTTP status code to the response.
```